### PR TITLE
Titlecase synced table column names in filters tab

### PR DESCRIPTION
**Pull request by fungjj92**
Synced:
screen shot 2016-08-24 at 9 56 20 am
screen shot 2016-08-24 at 9 55 59 am

Not synced table:
screen shot 2016-08-24 at 9 57 07 am

### CODE REVIEW

File: lib/assets/javascripts/cartodb/old_common/forms/filters_combo.js [hunk 1]
```diff
@@ -16,7 +16,13 @@
...
 
     _formatResult: function(data) {
-      return  '<span class="value">' + data.id + '</span>' + '<span class="type">' + (data.text && data.text.charAt(0)) + '</span>'
+      // Titlecase column name if synced table
+      var name = data.id.slice();
+      if (table.table.get('synchronization') && table.table.get('synchronization').from_external_source){
+        name = Sugar.String.titleize(name);
+      }
+
+      return  '<span class="value">' + name + '</span>' + '<span class="type">' + (data.text && data.text.charAt(0)) + '</span>'
```
Comment (flibbertigibbet) [inline lib/assets/javascripts/cartodb/old_common/forms/filters_combo.js:25]: Why test `charAt(0)` here? Does that return something different than `data.text.length`?

**Comment by flibbertigibbet** [comment]
I cannot test synchronized tables locally at the moment, but code LGTM.

File: lib/assets/javascripts/cartodb/old_common/forms/filters_combo.js (hunk 1 above)
Comment (fungjj92) [reply lib/assets/javascripts/cartodb/old_common/forms/filters_combo.js:25]: Great question! This was there already, shrug.

//update: I tested it, and it is necessary for the number/string indicator

Comment (flibbertigibbet) [reply lib/assets/javascripts/cartodb/old_common/forms/filters_combo.js:25]: ok cool

**Pull request merged**